slack/commands.py

```diff
@@ -287,6 +287,13 @@ def command_slack_reply(buffer: str, args: List[str], options: Options):
         run_async(slack_buffer.post_message(split_args[1], thread_ts, broadcast))
 
 
+@weechat_command("away|active")
+def command_slack_presence(buffer: str, args: List[str], options: Options):
+    slack_buffer = shared.buffers.get(buffer)
+    if slack_buffer:
+        run_async(slack_buffer.workspace.api.set_presence(args[0]))
```

Comment:
So like this:

```suggestion
    if slack_buffer is None:
        return

    new_presence = args[0]
    if new_presence != "active" and new_presence != "away":
        print_error(
            f'Error with command "/slack presence {args[0]}" (help on command: /help slack presence)'
        )
        return

    run_async(slack_buffer.workspace.api.set_presence(new_presence))
```

Reply:
Thanks! Hopefully this is the right one.

Reply:
I pushed a change to use the `new_presence` variable in the call to `set_presence`, otherwise the type narrowing doesn't (which is why the CI build failed).